woocommerce/google-listings-and-ads
# Add Reports' sub navigation.

### Changes proposed in this Pull Request:

Adds Reports' sub-navigation, using existing `.subsubsub` class, to share styling with the existing component written with jQuery.

Closes #98.


### Screenshots:


![Screenshoot for tab navigation and sub navigation.](https://user-images.githubusercontent.com/17435/107673135-fddd5580-6c95-11eb-9950-527281db3f6a.png)



### Detailed test instructions:

1. `git checkout feature/98-reports-nav && npm install && npm run build`
2. Go to [/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Freports%2Fprograms](http://localhost/wordpress/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Freports%2Fprograms)
3. Check if the look and feel matches Figma designs, requirements at https://github.com/woocommerce/google-listings-and-ads/issues/98, and is consistent with other 3rd-level navigation cases, for example, **WooCommerce > Orders**


### Changelog Note:

none

### Doubts:

1. The look does not match Fgima pixel perfect, but I tried to make it match the existing legacy component, pulse use defined `main-gap` for margins for consistency.
2. I wonder whether I should rely on existing styles for `.subsubsub` defined in woocommerce, or should I copy them all. I decided to re-use them for better consistency, DRY and YAGNI.
